ebuild-writing/using-eclasses/text.xml

```diff
@@ -25,6 +25,15 @@ illegal (except where the inheritance criteria are cache-constant <d/> see <uri
 link="::general-concepts/portage-cache"/>).
 </p>
 
+<p>
+When using <c>inherit</c>, it is best practice to sort the arguments (eclasses)
+alphabetically. An exception is where the phases exported by an eclass are
+affected by subsequent arguments. For example, <c>multilib-minimal.eclass</c>
+mentions in its
+<uri link="::eclass-reference/multilib-minimal.eclass/">documentation</uri>
+that it should be inherited last because it overrides most phases.
+</p>
```

Comment:
How about something similar to the following instead?

```suggestion
<p>
When using <c>inherit</c>, it is best practice to sort the arguments (eclasses)
alphabetically.
</p>

<important>
Don't rely on inherit order. It can be different from what it appears to be
because eclasses can indirectly inherit each other. If several eclasses export
the same phase functions, the ebuild should therefore define that function
and explicitly call the eclass functions in a well defined order.
</important>
```

Reply:
Again, disagree, noone is going to clutter multilib ebuilds like this.

Reply:
"This eclass must always go last" is strange advice. What if more than one eclass says that?

Reply:
The eclass won't say it, it's the use that implies where it does (even though anything overriding `multilib-minimal` would seem strange)

Reply:
But it does:
```
# multilib-minimal should _always_ go last in inherit order!
```